### ISSUE TITLE
Support self managed ConfigMapCommon

### ISSUE THREAD

**Issue by qixiaogang** [Altinity/clickhouse-operator#357]
We are using clickhouse-operator to set up clickhouse cluster across kube clusters. To achieve this, we have an operator to create and update common configmap, because we need to put all pod  accross different kube clusters into one remote_servers.xml, and ip is used in remote_servers.xml. When pod ip changed, we need to update the remote_servers.xml. 

common configmap is configmap `configMapCommonNamePattern = "chi-" + macrosChiName + "-common-configd"` in clickhouse operator.

So we want to support feature: ability to disable create and update common configmap in clickhouse operator. So we can self manage common configmap.

**Comment by qixiaogang** [comment]
@alex-zaitsev  Can you help to check above request?

**Comment by alex-zaitsev** [comment]
@qixiaogang , it is scheduled for development as we discussed on the call.

**Comment by qixiaogang** [comment]
@alex-zaitsev  thanks

**Comment by qixiaogang** [comment]
@alex-zaitsev May I ask the progress for this feature? Thanks in advance.

**Comment by alex-zaitsev** [comment]
@qixiaogang , we are working on configuration changes in scope of 0.11 release and plan to address your case there. 
Why do you require to manage config map directly?
Would it be sufficient to provide custom remote_servers in CHI resources? You can do it right now using 'files' section of configuration, but it will restart the pods. We may have a dedicated section for changes with no restart. That will be merged to the common configmap.

**Comment by qixiaogang** [comment]
@alex-zaitsev  We develop a across AZ clickhouse cluster, and we want to manage the remote_servers.xml ourself (put all servers from multi AZs together). 
Have a dedicated section for changes with no restart will also work for us.

**Comment by alex-zaitsev** [comment]
@qixiaogang, Implemented in 0.11.0 using custom config files. You can supply remote_servers override as follows:
```
configuration:
 files:
    config.d/remote_servers.xml:
        <yandex>
           <remote_servers replace="1">
           <!-- put your server configuration here -->
           </remote_servers>
        </yandex>
```
